pkg/cluster/config.go

```diff
@@ -296,3 +297,18 @@ func ValidateCIDR(cidr string) error {
 	}
 	return nil
 }
+
+// GetClusterDNS returns the IP address of the DNS server, given the service CIDR of the cluster
+func GetClusterDNS(ipRange string) (string, error) {
```

Comment:
I should have caught this yesterday. This should be `GetDNS`, so that the call site reads `cluster.GetDNS` rather than `cluster.GetClusterDNS`.

Reply:
Well, leave that to me; "I should have caught this yesterday".  Stuttering Alert!.